### PR TITLE
story: UI update for AVL publisher flow to include new static IP addresses

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: config/.env.local.template [hunk 1]
```diff
@@ -66,6 +66,7 @@ CAVL_CONSUMER_URL=__change_me__
 # Central AVL Service
 # ------------------------------------------------------------------------------
 CAVL_API_BASE_URL=__change_me__
+CAVL_IP_ADDRESS_LIST=
```
Comment (nbetts) [inline config/.env.local.template:69]: where are the IP addresses set if this is empty (and the other config templates)?

Comment (alainajaffer) [reply config/.env.local.template:69]: prod ones haven't been determined yet (correct me if I am wrong @DeannaSharma and @ljones92  ) so I assume this value will be added in the .env value hardcoded and provided based on the environment running.

**Review by nbetts — APPROVED** [review]

File: config/.env.local.template (hunk 1 above)
Comment (DeannaSharma) [reply config/.env.local.template:69]: Yeah prod IPs haven't been determined. 

I see what Nathan is getting at, if whatever reason these environment variable is not set what behaviour do we want to see? 

Maybe we need a check in `forms.py` where if `ALLOW_LIST_IP_ADDRESSES` is undefined then we do not show the bit of text on the front end about white listing IPs.

Comment (alainajaffer) [reply config/.env.local.template:69]: done

**Pull request merged**